logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/FeatureConfigEventReceiver.kt

```diff
@@ -84,7 +86,8 @@ internal class FeatureConfigEventReceiverImpl internal constructor(
     private suspend fun handleFeatureConfigEvent(event: Event.FeatureConfig): Either<CoreFailure, Unit> =
         when (event) {
             is Event.FeatureConfig.FileSharingUpdated -> fileSharingConfigHandler.handle(event.model)
-            is Event.FeatureConfig.MLSUpdated -> mlsConfigHandler.handle(event.model)
+            is Event.FeatureConfig.MLSUpdated -> mlsConfigHandler.handle(event.model, false)
+            is Event.FeatureConfig.MLSMigrationUpdated -> mlsMigrationConfigHandler.handle(event.model, false)
```

Comment:
suggestion: add arg name for 'false' value.

Reply:
Done